toolbase.js

```diff
@@ -1582,10 +1582,14 @@ function lookupGetRegExp(matchingValue, lookupName, matchingRegExpColumn, column
 	while(lookupItems.hasNext()) {
 		const lookupItem = lookupItems.getNext();
 		var regex = lookupItem.get(matchingRegExpColumn);
-		if(regex) {
+		if(!regex instanceof RegExp) {
 			if(matchingValue.match(new RegExp(regex, 'ig'))) {
 				return lookupItem.get(columnToRetrieveValueFrom);
 			}
+		} else {
+			if(matchingValue.match(regex)) {
```

Comment:
here check if the matchingValue has the function "match", it might not be a string at all
For that you can use if(matchingValue.match && matchingValue.match(.....

Reply:
Ah, cool, stimmt, damit kann ich checken obs ein string ist :-)